src/main/resources/swagger/api-docs.yaml

```diff
@@ -427,6 +427,39 @@ paths:
           description: Workspace or entity type does not exist
         500:
           description: Internal Server Error
+  /workspaces/{workspaceNamespace}/{workspaceName}/entities/{entityType}/export:
```

Comment:
I would prefer the endpoint `/tsv` or `/as-tsv` rather than `/export`. To me, "export" sounds like you're kicking off an asynchronous task to export the data somewhere else.